src/components/ToggleContent.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { font, springs } from '@aragon/ui'
+import { Transition, animated } from 'react-spring'
+
+import arrow from '../assets/arrow.svg'
+
+export default class ToggleContent extends React.Component {
+  static propTypes = {
+    labelOpen: PropTypes.string.isRequired,
+    labelClosed: PropTypes.string.isRequired,
+    children: PropTypes.node.isRequired,
+  }
+
+  state = { opened: false }
+  handleClick = () => {
+    this.setState(({ opened }) => ({
+      opened: !opened,
+    }))
+  }
+  render() {
+    const { opened } = this.state
+    const { labelOpen, labelClosed, children } = this.props
+    return (
+      <div>
+        <Label onClick={this.handleClick}>
+          {opened ? labelOpen : labelClosed} <Arrow opened={opened} />
```

Comment:
For consistency, we now point the array in the direction of the action to be taken. This was just introduced for the system apps menu dropdown https://github.com/aragon/aragon/blob/9334b199d5a7962bfe4456aea3a31e5327a726d8/src/components/MenuPanel/MenuPanel.js#L200-L205